app/views/customers/_list_body.html.slim

```diff
@@ -0,0 +1,20 @@
+tbody
+  - @customers.each do |user|
+    tr
+      td
+        = user.personal_info.first_name
+      td
+        = user.primary_phone_no
+      td
+        = user.email
+      td
+        = user.personal_info.date_of_birth
+      - user.address.each do |a|
+        td
+          = a.city if a.address_type == 'permanent'
```

Comment:
@RaviMariya-BTC : Why permanent is hardcoded? we have used it model by creating enum?